### ISSUE TITLE
Cant Create Album or Upload

### ISSUE THREAD

**Issue by cheesewizz** [Piwigo/Piwigo#806]
error message:
SyntaxError: JSON.parse: unexpected character at line 1 column 1 of the JSON data piwigo

I am using Centos7 

    Server: Localhost via UNIX socket
    Server type: MySQL
    Server version: 5.6.26 - MySQL Community Server (GPL)
    Protocol version: 10
    User: root@localhost
    Server charset: UTF-8 Unicode (utf8)

    Apache
    Database client version: libmysql - 5.6.26
    PHP extension: mysqli Documentation
    PHP version: 5.6.29

Thanks

**Comment by svaens** [comment]
I also see this problem. Any progress? Any work-around?

**Comment by webmind** [comment]
I got this problem, can't upload via webform. It just gives me a bar which doesn't progress. Getting this error in de firefox debug screen:
```
SyntaxError: JSON.parse: unexpected character at line 1 column 1 of the JSON data
jquery.min.js:5:15991
TypeError: uploadCategory is null
```

 

Running the latest version as of today.

**Comment by JasonZX12R** [comment]
Same here - I have had multiple issues with the Piwigo installation.  Is this still being actively supported?

**Comment by flop25** [comment]
yes it is ; you will easily see this on our forums where you can many informations such as answers or a community

**Comment by dwarfmaster** [comment]
I have the same problem (nothing shows on firefox console). Has there been any progress on that issue ?

**Comment by dwarfmaster** [comment]
I fixed it ! I was also having the same problem as #1134 and I was just ignoring it, but it turns out that it was the cause : php was outputing the warning also to the json output, causing the parse to fail. I fixed it by making sure the `display_errors` was set to false in my `php.ini`, and by editing `include/common.inc.php:88` which was setting `display_errors` to true.
